### ISSUE TITLE
[BUG] rax40: socket files in /tmp are not deleted

### ISSUE THREAD

**Issue by rmelotte** [prplfoundation/prplMesh#1143]
## Description
After a few weeks of running certification tests, deploying prplMesh on the device failed with the following error:
```
Collected errors:
* pkg_init_from_file: Failed to make temp file /tmp/opkg-DDbfJH/prplmesh.ipk.control.XXXXXX: No space left on device.
```
After a quick inspection, there is still a lot of space left on the device, but most probably it ran out of inodes.

The `/tmp` directory contains almost 60.000 socket files named like `wpa_ctrl_X_Y`.

Example of a test where the issue occur: https://gitlab.com/prpl-foundation/prplMesh/-/jobs/509460376

**Comment by arnout** [comment]
I've taken a very quick look at wpa_ctrl.c and dwpal. The `wpa_ctrl_...` files are unlinked in wpa_ctrl_close(), which gets called from the dwpal-bwl destructor. So it seems that that destructor doesn't get called on program termination.

However, when I manually kill the agent, I see in the log files that the destructor *does* get called. So I'm not sure what is happening on the RAX40...

**Comment by rmelotte** [comment]
I just checked both on the Glinet device and the Turris Omnia we use to run the "status tests", and they also have a lot of those socket files left behind.

**Comment by RanRegev** [comment]
I am looking at this. [feature/issue-1143](https://github.com/prplfoundation/prplMesh/tree/feature/issue-1143)

**Comment by arnout** [comment]
Should be fixed by #1454.
